view.php

```diff
@@ -116,23 +122,18 @@ function mod_tincanlaunch_launchexperience(registration) {
 
 if ($lrsrespond == 200) {
     $registrationdatafromlrs = json_decode($getregistrationdatafromlrsstate->content->getContent(), true);
-    if ($tincanlaunch->tincanmultipleregs) {
-        echo "<p id='tincanlaunch_newattempt'><a onclick=\"mod_tincanlaunch_launchexperience('"
-            .$registrationid
-            ."')\" style=\"cursor: pointer;\">"
-            .get_string('tincanlaunch_attempt', 'tincanlaunch')
-            ."</a></p>";
-    }
```

Comment:
Why are you removing this section?

Reply:
Oh I see. You moved it. How does this affect the UI? If it's different, can you share a screenshot?

Reply:
It's only different if you have previous attempts. Otherwise it shows up like it use to. Reason being, is because we don't have a warning or any text with the 'launch new attempt' so a blind user will only hear 'launch attempt' and not hear/know about the table with their previous attempts. This way, the non-sighted user will be able to hear about the previous attempts before hearing launch new attempt.

Reply:
![after attempt](https://user-images.githubusercontent.com/3696698/41057784-07a4c7e8-6996-11e8-9065-fdab1b60a219.png)
![before attempt](https://user-images.githubusercontent.com/3696698/41057787-0a7b4d5c-6996-11e8-9fbe-580f0d5ca7d0.png)


FYI, I have to truncate some of the information out, due to the fact I'm working on another project with the plugin. So sorry in advance for a super small glimpse of the page.

Reply:
If you haven't done an attempt or it's the return home link. It will look the same as before.